kubernetes/resource_kubernetes_namespace_test.go

```diff
@@ -352,6 +406,20 @@ resource "kubernetes_namespace" "test" {
 `, nsName)
 }
 
+func testAccKubernetesNamespaceSpecConfig(nsName string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_namespace" "test" {
+  metadata {
+    name = %[1]q
```

Comment:
Usually we use `%s` for the name. Is there a reason to use this other notation? I'm not familiar with it offhand.

```suggestion
    name = "%s"
```

Reply:
We use a mixture of `%q` and `%s` in the test fixtures, so we aren't consistent about it. The value of using the `%q` verb is just to save a few characters, particularly when you have to escape the quotes around it. The square brackets denote which argument to use. In this case it is superfluous as there is only one argument.

Side note: I like to use [gore](https://github.com/motemen/gore) to figure out language features quickly when I don't want to read through documentation:

```go
gore> :import fmt
fmt.Sprintf("%s %q %[1]q", "first", "second", "third")
"first \"second\" \"first\""
```